test/linalg/symmetric.jl

```diff
@@ -129,6 +129,12 @@ let n=10
         @test det(a + a.') ≈ det(Symmetric(a + a.', :U))
         @test det(a + a.') ≈ det(Symmetric(a + a.', :L))
 
+        # isposdef
+        if eltya != BigFloat # isposdef not defined for Matrix{BigFloat}
+            @test isposdef(Symmetric(asym)) == isposdef(full(Symmetric(asym)))
```

Comment:
Might be good to directly test the methods for the mutating version, `isposdef!`, that you've added here.